js/core/webidl-oldschool.js

```diff
@@ -118,7 +118,7 @@ define(
         WebIDLProcessor.prototype = {
             setID:  function (obj, match) {
                 obj.id = match;
-                obj.refId = obj.id.replace(/[^a-zA-Z_\-]/g, "");
+                obj.refId = obj.id.replace(/[^a-zA-Z0-9_\-]/g, "");
```

Comment:
Good point! I don't see a good way to do it with only once `replace` so I'm just using 2.